### PR TITLE
Brahim

### PR DESCRIPTION
**Pull request by brahimkedjar**

### CODE REVIEW

File: database/migrations/2022_11_11_191013_modules.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('Modules', function (Blueprint $table){
+            $table->id();
+            $table->unsignedBigInteger('users_id')->nullable();
```
Comment (nminfo86) [inline database/migrations/2022_11_11_191013_modules.php]: change this to user_id ,level_id, speclalities_id without "s"

File: database/migrations/2022_11_11_192021_faculties.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('faculties', function (Blueprint $table){
+            $table->id();
+            $table->string('faculte')->nullable();
```
Comment (nminfo86) [inline database/migrations/2022_11_11_192021_faculties.php]: change this to faculty not faculte

**Review by nminfo86 — CHANGES_REQUESTED** [review]
Please see comments before merge to main branch

File: database/migrations/2022_11_11_192021_faculties.php (hunk 2 above)
Comment (brahimkedjar) [reply database/migrations/2022_11_11_192021_faculties.php]: done

File: database/migrations/2022_11_11_191013_modules.php (hunk 1 above)
Comment (brahimkedjar) [reply database/migrations/2022_11_11_191013_modules.php]: done

**Review by nminfo86 — APPROVED** [review]
ya3ik saha

**Pull request merged**